Effects/Parallax/source/Flake.hx

```diff
@@ -0,0 +1,74 @@
+package;
+
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.math.FlxMath;
+import flixel.math.FlxRandom;
+import flixel.util.FlxColor;
+
+class Flake extends FlxSprite
+{
+	private var _level:Int = 0;
+	private var _initialized:Bool = false;
+	
+	
+	public function new(Level:Int = 0) 
+	{
+		
+		super();
+		_level = Level;
+		revive();
+		_initialized = true;
+	}
+	
+	override public function revive():Void 
+	{
+		var size:Int = Std.int(FlxMath.bound(_level / 2, 1, 2));
+		makeGraphic(size, size, FlxColor.WHITE.getDarkened(.5 - (_level * .06)));
+		super.revive();
+		alpha = 1;
+		if (_initialized)
+		{
+			x = FlxG.random.int(0, FlxG.width * 4 * _level);
+			y = FlxG.random.int( -5, -10);
+		}
+		else
```

Comment:
`_initialized` is used to tell the difference between "being created before the game actually starts" and "being recycled".
If it's `FALSE`, then we can put the flakes anywhere, randomly, on the screen.
Once the game starts though, and it's set to `TRUE`, then we only want them to spawn 'off-screen'. This makes it so that the flakes don't wait for the game to start before starting to fall, and/or so they don't spawn in the middle of the screen once the game has started. There may be better methods, but this works well enough.